Bridge/DoctrineStatsBundle/DataCollector/DoctrineStatsCollector.php

```diff
@@ -2,10 +2,10 @@
 
 namespace steevanb\DoctrineStats\Bridge\DoctrineStatsBundle\DataCollector;
 
+use Doctrine\Common\Persistence\ManagerRegistry;
```

Comment:
`Doctrine\Common\Persistence\ManagerRegistry` should be remplaced by `Doctrine\Persistence\ManagerRegistry` and a conflit section should be add for doctrine/persistance < 1.3

Reply:
I totally missed that comment, and that's correct, now that Doctrine 3 starts to be released it is crashing. I fixed it.